packages/pipewire/pipewire.pacscript

```diff
@@ -15,6 +15,6 @@ build() {
 }
 
 install() {
-    cd builddir/
+    cd builddir/ || exit 1
```

Comment:
```suggestion
    cd builddir/ || return 1
```

If we use `return` then Pacstall can handle the error code properly instead of the program stopping execution at this point.